scalableminds/webknossos
# Remove default teams from datasets

### Mailable description of changes:
 - Datasets no longer have a default team
 - TeamManager and Admins can import datasets and assign allowed teams

### Steps to test:
- upload new dataset -> should not have a default team
- import dataset -> should be able to assign allowed teams
- edit dataset -> should be able to edit allowed teams

### Issues:
- fixes #1533 

------
